examples/react-web/src/lobby/lobby.js

```diff
@@ -28,8 +28,8 @@ const LobbyView = () => (
     <h1>Lobby</h1>
 
     <Lobby
-      gameServer="localhost:8000"
-      lobbyServer="localhost:8001"
+      gameServer="http://localhost:8000"
```

Comment:
I've just made a fresh clone of master and the bug still happens... (since commit a285fbf0dea0c8a2f2c27f43361f967ab3d28a8d actually) Does 'create a room' work for you? 


![lobby](https://user-images.githubusercontent.com/1760098/51792613-cafdbd00-21b3-11e9-92b0-1e24d9741aa6.png)

Reply:
Ah yes, I see the problem now (I think I may have been synced to a commit before that).

Reply:
Strange, that worked for me..